tekton/resources/nightly-tests/pipeline-deploy-test-s390x-template.yaml

```diff
@@ -69,12 +64,43 @@ spec:
         - name: remote-port
         - name: remote-user
         tasks:
+        - name: git-clone-plumbing
+          taskRef:
+            name: git-clone
+          params:
+          - name: url
+            value: https://github.com/tektoncd/plumbing
+          - name: revision
+            value: main
+          - name: subdirectory
+            value: src/github.com/tektoncd/plumbing
+          workspaces:
+          - name: output
+            workspace: shared-workspace
+            subPath: source-code
+        - name: git-clone-pipeline
+          runAfter: [git-clone-plumbing]
```

Comment:
I guess they cannot run in parallel because the write to the same volume?

Reply:
yes, correct